imcsdk/imcdriver.py

```diff
@@ -75,7 +74,7 @@ def connect(self):
 
         # This is the only difference; default wrap_socket uses SSLv23
         self.sock = ssl.wrap_socket(sock, self.key_file, self.cert_file,
-                                    ssl_version=ssl.PROTOCOL_TLSv1)
+                                    cert_reqs=ssl.CERT_NONE, ssl_version=ssl.PROTOCOL_TLSv1_2)
```

Comment:
This would need some testing across various IMC releases and Hardware combinations. Let me check who can help with this.

Reply:
Ok, thanks.  My home lab is pretty dated... No M5 or higher servers.